shared/common-adapters/user-card.native.js

```diff
@@ -12,10 +12,12 @@ export default class UserCard extends Component<void, Props, void> {
     const url = this.props.username ? `https://keybase.io/${this.props.username}` : null
     return (
       <Box style={{...styles.container, ...this.props.outerStyle}}>
+        <Box style={styles.avatar}>
```

Comment:
simpler to just wrap this in a box vs using absolute positioning. just leveraging flexbox seems to produce stabler results